tests/test_util/test_util_inventory.py

```diff
@@ -1,7 +1,10 @@
 """Test inventory util functions."""
+from __future__ import annotations
```

Comment:
And don't forget this one in other test files.

Reply:
ruff should lint any files that use modern type syntax and forget to add `from __future__ import annotations`